docs/src/snap/howto/epa.md

```diff
@@ -0,0 +1,1061 @@
+
+{{product}} is delivered as a 
+[snap](https://documentation.ubuntu.com/canonical-kubernetes/latest/snap/).
+
+This section explains how to set up a dual node {{product}} cluster for testing
+EPA capabilities.
+
+### Control plane and worker node 
+
+1. [Install the
+   snap](https://documentation.ubuntu.com/canonical-kubernetes/latest/snap/howto/install/snap/)
+   from the moonray track. The [beta
+   channel](https://documentation.ubuntu.com/canonical-kubernetes/latest/snap/explanation/channels/)
+   is used at this point as the end configuration of the k8s snap is not
+   finalised yet.
+
+```
+sudo snap install k8s --classic --channel=1.30-moonray/beta
+```
+
+2. Create a file called *configuration.yaml*. In this configuration file we let
+   the snap start with its default CNI (calico), with CoreDNS deployed and we
+   also point k8s to the external etcd. 
+
+```
+cluster-config:
+  network:
+    enabled: true
+  dns:
+    enabled: true  local-storage:
+    enabled: true
+extra-node-kubelet-args:
+  --reserved-cpus: "0-31"
+  --cpu-manager-policy: "static"
+  --topology-manager-policy: "best-effort"
+```
+
+3. Bootstrap {{product}} using the above configuration file.
+
+```
+sudo k8s bootstrap --file configuration.yaml
+```
+
+#### Verify control plane node is up
+
+After a few seconds you can query the API server with:
+
+```
+sudo k8s kubectl get all -A
+```
+
+### Second k8s node as worker 
+
+1. Install the k8s snap on the second node
+
+```
+sudo snap install k8s --classic --channel=1.30-moonray/beta
+```
+
+2. On the control plane node generate a join token to be used for joining the
+   second node
+
+```
+sudo k8s get-join-token --worker
+```
+
+3. On the worker node create the configuration.yaml file
+
+```
+extra-node-kubelet-args:
+  --reserved-cpus: "0-31"
+  --cpu-manager-policy: "static"
+  --topology-manager-policy: "best-effort"
+```
+
+4. On the worker node use the token to join the cluster
+
+```
+sudo k8s join-cluster --file configuration.yaml <token-generated-on-the-control-plane-node>
+```
+
+
+### Verify the two node cluster is ready 
+
+After a few seconds the second worker node will register with the control
+plane. You can query the available workers from the first node:
+
+```
+sudo k8s kubectl get nodes
+```
+
+The output should list the connected nodes:
+
+```
+NAME          STATUS   ROLES                  AGE   VERSION
+pc6b-rb4-n1   Ready    control-plane,worker   22h   v1.30.2
+pc6b-rb4-n3   Ready    worker                 22h   v1.30.2
+```
+
+### Multus and SRIOV setup 
+
+Get the thick plugin (in case of resource scarcity we can consider deploying
+the thin flavor)
+
+```
+sudo k8s kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/master/deployments/multus-daemonset-thick.yml
+```
+
+Note: the memory limits for the multus pod spec in the DaemonSet should be
+increased (i.e. to 500Mi instead 50Mi) to avoid OOM issues when deploying
+multiple workload pods in parallel.
```

Comment:
```suggestion
```{note} 
The memory limits for the Multus pod spec in the DaemonSet should be
increased (i.e. to 500Mi instead 50Mi) to avoid OOM issues when deploying
multiple workload pods in parallel.
```
Please add the end tics. Capitalize Multus to be consistent in the doc.

Reply:
:+1: